product_service_duration/demo/calendar_event_demo.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="calendar_event_1" model="calendar.event">
+        <field name="name">Client Consultation</field>
+        <field name="start" eval="time.strftime('%Y-%m-10 12:00:00')" />
+        <field name="stop" eval="time.strftime('%Y-%m-10 17:00:00')" />
+        <field name="duration" eval="5.00" />
+        <field name="template_ids"
+               eval="[(4, ref('product_service_duration.product_template_1')),
+                      (4, ref('product_service_duration.product_template_2'))]" />
```

Comment:
Yup, that's actually what you want to do. Subsequent updates of this record will actually continue to insert the same records into the m2m, because the old ones aren't being removed. `6` is just a `5` followed by a `4` for each `id` in `ids`